docs/testrunner-toolkit/configuration/xcuitest.md

```diff
@@ -1,190 +1,468 @@
+<p><small>| REQUIRED | STRING |</small></p>
 
-__Example__:
+The directory path to the testing application. The default directory is `{project-root}/apps/testfile.app`, and the property supports expanded environment variables to designate the path, as shown in the following examples.
 
 ```yaml
-        platformVersion: 14.*
+  testApp: ./apps/SwagLabsMobileAppUITests-Runner.app
 ```
 
-### `options`
+```yaml
+  testApp: $TEST_APP
+```
+---
 
-__Description__:  Further specify certain device attributes within the pool of devices that match the `name` and `version` criteria.
+## `suites`
+<p><small>| REQUIRED | OBJECT |</small></p>
 
-__Type__: *object*
+The set of properties providing details about the test suites to run. May contain multiple suite definitions. See the full [example config](#example-configuration) for an illustration of multiple suite definitions.
 
-#### `carrierConnectivity`
+:::note Real Devices Only
+At this time, `saucectl` does not support automated tests running on simulators.
+:::
+---
 
-__Description__: Request that the matching device is also connected to a cellular network.
+### `name`
+<p><small>| REQUIRED | STRING |</small></p>
+
+The name of the test suite, which will be reflected in the results and related artifacts.
+
+```yaml
+  - name: "saucy test"
+```
+---
 
-__Type__:  *bool*
+### `env`
+<p><small>| OPTIONAL | OBJECT |</small></p>
 
-__Example__:
+A property containing one or more environment variables that may be referenced in the tests for this suite. Expanded environment variables are supported.
 
 ```yaml
-       carrierConnectivity: true
+  env:
+    hello: world
+    my_var: $MY_VAR
+```
+---
+
+### `devices`
+<p><small>| REQUIRED | OBJECT |</small></p>
+
+The parent property that defines how to select real devices on which to run the test suite. You can request a specific device using its ID, or you can specify a set of criteria to choose the first available device that matches the specifications.
+
+When an ID is specified, it supersedes the other settings.
+
+```yaml
+ddevices:
+  - name: "iPhone 11"
+    platformVersion: "14.3"
+  - id: iPhone_11_14_5_real_us
+```
+---
+
+#### `id`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Request a specific device for this test suite by its ID. You can look up device IDs in the Sauce Labs app or using our [Get Devices API request](https://docs.saucelabs.com/dev/api/rdc#get-devices).
 
+```yaml
+        id: iPhone_11_14_5_real_us
 ```
+---
 
-#### `deviceType`
+#### `name`
+<p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__:  Request that the matching device is a specific type of device. Values:  `ANY`, `TABLET`, or `PHONE`.
+Find a device for this test suite that matches the device name or portion of the name, which may provide a larger pool of available devices of the type you want.
 
-__Type__:  *string*
+```yaml title="Use Complete Name"
+      - name: iPhone 11
+```
 
-__Example__:
+```yaml title="Use Pattern Matching"
+        name: iPhone*
```

Comment:
Matching in deviceName is real regular expression, so it needs to be `.*` (`.` behind a wildcard character, `*` meaning the repetition of the wilcard character can be from 0 to infinite).

```suggestion
        name: iPhone.*
```